cmd/antrea-agent/options.go

```diff
@@ -105,9 +105,17 @@ func (o *Options) validate(args []string) error {
 			return fmt.Errorf("Mode %s requires AntreaProxy to be enabled", o.config.TrafficEncapMode)
 		}
 		if o.config.EnableIPSecTunnel {
-			return fmt.Errorf("IPSec tunnel may only be enabled on %s mode", config.TrafficEncapModeEncap)
+			return fmt.Errorf("IPsec tunnel may only be enabled on %s mode", config.TrafficEncapModeEncap)
 		}
 	}
+	if o.config.NoSNAT && encapMode != config.TrafficEncapModeNoEncap {
+		return fmt.Errorf("NoSNAT is applicable to only %s mode", config.TrafficEncapModeEncap)
```

Comment:
"noSNAT is only applicable to the %s mode"

Reply:
Done.